### ISSUE TITLE
User doesn't get verification email after sign up

### ISSUE THREAD

**Issue by koonchaya** [rfcx/engineering-support#522]
Original report https://rfcx.slack.com/archives/C01BW5M71JS/p1710858515950369
https://rfcx.slack.com/archives/C01BW5M71JS/p1710862312671529
https://rfcx.slack.com/archives/C03FD1WD02J/p1711037990058799
## Background
User does not get the verification email after they signed up RFCx account. There are sometimes that 
1. User never gets the email even in spam or trash.
2. User gets the email after a while.

## Workaround
1. I resend the verification email from auth0 and user gets it but sometimes not. 
2. Manually verify the account if user never get the email but I made sure that user user the same email address to report this. 

Can we check why sometimes user doesn't get the verification email?

**Comment by koonchaya** [comment]
@RatreeOchn Can you bring this to EC on next monday?

**Comment by RatreeOchn** [comment]
> [@RatreeOchn](https://github.com/RatreeOchn) Can you bring this to EC on next monday?

Sure.

**Comment by rassokhin-s** [comment]
@koonchaya I checked the following things:
- Auth0 user registration flow including additional hooks / pipes
- Email provider configuration
Everything above looks correct and there are no any potential places for the problem to exist.

I also tried two times to sign up with an email address which has same domain as users had (Arbimon.Test@rspb.org.uk) and I saw in the email provider logs that the email has been sent.

According to this [article](https://community.auth0.com/t/email-received-with-delay/88043) all the points are valid for us except the last one - it seems that these users come from the org which uses custom email provider, so maybe it has some crazy spam rules and doesn't allow emails to come.

I can't find any useful logs for these user cases as it's been almost two weeks since that. I suggest to close this task for now and check the issue immediately if it happens again to be able to find some useful logs.
